### PR TITLE
[3.2] Merge Category and Property fields in the Project Settings

### PR DESCRIPTION
**Pull request by dalexeev**
Closes godotengine/godot-proposals#1545 for 3.2.

Please note that, unlike #45716, this PR is quite small, so I would like it to be merged before 3.2.4 is released. :smiley:

### CODE REVIEW

**Review by Calinou — APPROVED** [review]

File: editor/project_settings_editor.cpp [hunk 1]
```diff
@@ -875,7 +871,7 @@ void ProjectSettingsEditor::_item_add() {
 
 	undo_redo->commit_action();
 
-	globals_editor->set_current_section(catname);
+	globals_editor->set_current_section(name.split("/")[0]);
```
Comment (dalexeev) [inline editor/project_settings_editor.cpp:874]: 4.0 uses `name.get_slice("/", 1)` instead. What's better?

Comment (akien-mga) [reply editor/project_settings_editor.cpp:874]: `get_slice()` is better, the other one can crash if there's no result.

Comment (dalexeev) [reply editor/project_settings_editor.cpp:874]: It seems to me that `string.split("/")` returns a `Vector<String>` containing at least one element. Ok, I'll fix it.

Comment (akien-mga) [reply editor/project_settings_editor.cpp:874]: Ah you're right, both are fine then.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**